store/copr/batch_coprocessor.go

```diff
@@ -882,7 +975,7 @@ func buildBatchCopTasksCore(bo *backoff.Backoffer, store *kvStore, rangesForEach
 			logutil.BgLogger().Debug(msg)
 		}
 		balanceStart := time.Now()
-		batchTasks = balanceBatchCopTask(bo.GetCtx(), store, batchTasks, isMPP, ttl, balanceWithContinuity, balanceContinuousRegionCount)
+		batchTasks = balanceBatchCopTask(bo.GetCtx(), aliveStores, batchTasks, balanceWithContinuity, balanceContinuousRegionCount)
```

Comment:
There may be some potential problems here by pass `aliveStores` to `balanceBatchCopTask`, I think it should pass  the alive stores that are expected to be used. For example, if `close_replicas` is used, then the expected 
 stores to be used is just a subset of `aliveStores`, in this case, pass `aliveStores` here will make the balance scores less then expected, and fallback to `balanceContinuousRegionCount` strategy, which is actually not expected.